docs/hosting/scaling/execution-modes-processes.md

```diff
@@ -45,8 +45,11 @@ This is the default setting for n8n. When running n8n in the `regular` mode, eve
 
 ### Queue
 
-`Queue` mode is designed for handling very high workloads. In this mode you run multiple instances of n8n: one ‘main' instance coordinates, and other ‘worker' instances actually process the executions. You can also add dedicated instances to handle incoming webhooks.
+`Queue` mode is designed for handling high workloads. In this mode you run multiple instances of n8n: one main instance coordinates work, and other worker instances process the executions. You can also add dedicated instances to handle incoming webhooks.
 
 `Queue` mode requires more setup than `regular` mode, but provides great scalability and stability with minimal latency.
 
+!!! note "Binary data storage"
+	n8n doesn't support queue mode with binary data storage. If your workflows need to persist binary data, you can't use queue mode.
```

Comment:
This made me realize that we don't seem to have a dedicated documentation for binary data storage.

Basically, when handling what we call "binary data" (files) that can be large, this data stays in memory and takes up a lot of space.

For simply moving files (download and upload, for instance), this is unnecessary and could cause n8n to crash.

Binary data storage has 2 options: in memory (default) or filesystem (where n8n saves data to disk instead, saving memory space).

I think we need to add this to our docs somewhere.

![image](https://user-images.githubusercontent.com/219272/212063754-d6d3fefc-f80e-4d37-9322-f8ad0d532fbb.png)

The settings for this feature can be seen here

Reply:
Added a doc about binary data, and split out the env vars in the env vars doc to make them more discoverable